index.html

```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
+               <b>CredentialSchema2023</b> is for the validation of W3C Verifiable Credentials, based
+               on representing JSON Schema in a <a>verifiable credential</a>. The version of [[JSON-Schema]]
+               can be any version noted in the section on <a href="#json-schema-specifications">JSON Schema Specifications</a>.
+            </p>
+            <p>
+             <table class="simple">
+                <thead>
+                   <tr>
+                      <th>Property</th>
+                      <th>Description</th>
+                   </tr>
+                </thead>
+                <tbody>
+                   <tr>
+                      <td>id</td>
+                      <td>The constraints on the <code>id</code> property are listed in the Verifiable Credentials
+                      Data Model specification [[VC-DATA-MODEL]]. The value is expected to be a URL that identifies
+                      the credential schema associated with the <a>verifiable credential</a>.</td>
+                   </tr>
+                   <tr>
+                      <td>type</td>
+                      <td>The <code>type</code> property MUST be CredentialSchema2023</td>
+                   </tr>
+                </tbody>
+               </table>
+            </p>
+               An example of utilizing the VC Data Model's <code>credentialSchema</code>
+               is provided below:
+               <pre class="example highlight" title="Example CredentialSchema2023">
+               {
+                 "@context": [
+                   "https://www.w3.org/2018/credentials/v1",
+                   "https://www.w3.org/2018/credentials/examples/v1"
+                 ],
+                 "id": "http://example.edu/credentials/3732",
+                 "type": ["VerifiableCredential", "EmailCredential"],
+                 "issuer": "https://example.com/issuers/14",
+                 "issuanceDate": "2010-01-01T19:23:24Z",
+                 "credentialSubject": {
+                   "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+                   "email": "subject@example.com"
+                 },
+                 "credentialSchema": {
+                   "id": "https://example.com/schemas/email-credential-schema.json",
+                   "type": "CredentialSchema2023"
+                 }
+               }
+              </pre>
+            </p>
+            <p>
+              <pre class="example highlight" title="Example Email Credential Schema">
+              {
+                "@context": [
+                    "https://www.w3.org/2018/credentials/v1",
+                    "https://www.w3.org/2018/credentials/examples/v1"
+                ],
+                "id": "http://example.edu/credentials/3732",
+                "type": ["VerifiableCredential", "CredentialSchema2023"],
+                "issuer": "https://example.com/issuers/14",
+                "issuanceDate": "2010-01-01T19:23:24Z",
+                "credentialSubject": {
```

Comment:
Some explanation on how to go from JSON schema --> CredentialSchema2023 would be useful.

Reply:
see above comment